README.md

```diff
@@ -90,6 +90,7 @@ function MyApp() {
 |onSelectBbox|Function that is called when the bbox is selected by keypress handlers.|n/a| `(index) => alert('Select bounding box with index: ' + index)`                                                                                                                                                                                                                                                    |
 |<b>Bbox props</b>|
 |colorScheme|Set custom colors for bbox|```{ border?: string; borderSelected?: string; borderRelated?: string; borderStructured?: string; borderStructuredSelected?: string; borderHovered?: string; borderStructuredHovered?: string; background?: string; backgroundSelected?: string; backgroundHovered?: string; backgroundRelated?: string; backgroundStructured?: string; backgroundStructuredSelected?: string;}```| n/a                                                                                                                                                                                                                                                                                                               |
+| renderBbox              | Function for render custom Bbox component | ```{ left: string; width: string; height: string; top: string; disabled: boolean; related?: boolean; selected?: boolean; structured?: boolean; scale: number; colorScheme?: IColorScheme; selectionMode?: TreeBboxSelectionMode; onClick?(e: any): void; }``` | `(props) =>  ReactElement`                                                                                                                                                                                                                                                                               |
```

Comment:
Please also update the description:
- Function for render custom Bbox component -> Function for render**ing** custom Bbox component
- Third column should be `n/a`
- Fourth column should be:
```
 ```(props: { left: string; width: string; height: string; top: string; disabled: boolean; related?: boolean; selected?: boolean; structured?: boolean; scale: number; colorScheme?: IColorScheme; selectionMode?: TreeBboxSelectionMode; onClick?(e: any): void; }) => ReactElement```
```